tools/workspace-plugin/src/generators/react-component/index.ts

```diff
@@ -29,7 +29,12 @@ export default async function (tree: Tree, schema: ReactComponentGeneratorSchema
       stdio: 'inherit',
     });
 
-    execSync(`yarn workspace ${npmPackageName} test -t ${componentName}`, {
+    // This is used only for integration testing purposes on CI as jest disables snapshot updates by default
+    const forceSnapshotUpdate = Boolean(process.env.__FORCE_SNAPSHOT_UPDATE__);
```

Comment:
private env variable usage in order to pass test with snapshot updates on tooling integration test in CI